bukkit/src/main/java/net/skinsrestorer/bukkit/SkinsRestorer.java

```diff
@@ -560,5 +570,16 @@ public SkinsRestorerBukkitAPI() {
         public void applySkin(PlayerWrapper playerWrapper, IProperty property) {
             skinApplierBukkit.applySkin(playerWrapper.get(Player.class), property);
         }
+
+        @Override
+        public String getMessage(ISRForeign foreign, MessageKeyGetter key, Object... args) {
+            String message = localeManager.getMessage(foreign, key.getKey());
+
+            if (message.contains("{prefix}")) {
+                message = message.replace("{prefix}", localeManager.getMessage(foreign, Locale.PREFIX.getKey()));
```

Comment:
is Config.DisablePrefix taken into account here?

Reply:
That option is removed currently, but maybe it makes sense to add back. Will look into it.

Reply:
> That option is removed currently, but maybe it makes sense to add back. Will look into it.

well, if prefix can be empty (`PREFIX = ""` ) it should be no issue right?

Reply:
Well, yes, but they would have to change it in every single translation file.